wallet/wallet-cli-lib/src/config.rs

```diff
@@ -135,6 +135,22 @@ pub struct CliArgs {
     /// Enable running the wallet service without RPC authentication (assuming wallet RPC server is enabled)
     #[arg(long, conflicts_with_all(["wallet_rpc_password", "wallet_rpc_username", "wallet_rpc_cookie_file"]))]
     pub wallet_rpc_no_authentication: bool,
+
+    /// Optional RPC address
+    #[arg(long, conflicts_with_all(["wallet_rpc_password", "wallet_rpc_username", "wallet_rpc_cookie_file", "wallet_rpc_no_authentication", "wallet_rpc_bind_address", "node_rpc_address", "node_rpc_cookie_file", "node_rpc_username", "node_rpc_password"]))]
+    pub connect_to_rpc_wallet_address: Option<String>,
+
+    /// Path to the RPC cookie file. If not set, the value is read from the default cookie file location.
+    #[clap(long)]
+    pub connect_to_rpc_wallet_cookie_file: Option<String>,
+
+    /// RPC username (either provide a username and password, or use a cookie file. You cannot use both)
+    #[clap(long)]
+    pub connect_to_rpc_wallet_username: Option<String>,
+
+    /// RPC password (either provide a username and password, or use a cookie file. You cannot use both)
+    #[clap(long)]
+    pub connect_to_rpc_wallet_password: Option<String>,
```

Comment:
Please change `connect_to` to `remote` in all these parameters.

Reply:
renamed